.github/workflows/bindings.yml

```diff
@@ -52,19 +52,28 @@ jobs:
           cd crypto-ffi/bindings
           ./gradlew android:build -x lint -x lintRelease
 
-  check-swift:
+  build-swift:
     runs-on: macos-latest
+    strategy:
+      matrix:
+        task:
+          - name: ios-device
+            target: aarch64-apple-ios
+          - name: ios-simulator-x86
+            target: x86_64-apple-ios
+          - name: ios-simulator-arm
+            target: aarch64-apple-ios-sim
     steps:
       - uses: actions/checkout@v4
-      - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
           rustflags: ''
-          target: "aarch64-apple-ios,x86_64-apple-ios,aarch64-apple-ios-sim"
+          target: ${{ matrix.task.target }}
       - name: Setup cargo-make
         uses: davidB/rust-cargo-make@v1
       - run: |
           cd crypto-ffi
-          cargo make ios-create-xcframework
```

Comment:
Do we want a downstream job that does `cargo make ios-create-xcframework` and
```
- needs: [ios-device, ios-simulator-x86, ios-simulator-arm]
```
?

Reply:
Yeah, there's going to be another job that collects all of the artifacts. Note this is still a draft. :wink: